pkg/ocm/machines.go

```diff
@@ -254,7 +254,8 @@ func (mtl *MachineTypeList) ValidateMachineType(machineType string, multiAZ bool
 
 	if v == nil {
 		allMachineTypes := strings.Join(mtl.IDs(), " ")
-		err := fmt.Errorf("A valid machine type number must be specified\nValid machine types: %s", allMachineTypes)
+		err := fmt.Errorf("Machine type '%s' not found in region machine type list: %s",
```

Comment:
What about adding the region there too for even better clarity? Like
```
Machine type '%s' not found in region '%s'. Available machine type list: %s"
```

Reply:
Good idea, let me see if it's available in the call

Reply:
"not found in region" is slightly misleading. The problem is that the machine type isn't found in the specific availability zone within the region. For example, even though us-east-1 (the region) has lots of m6i machines, us-east-1e (the AZ) might not have any.

Reply:
Yes, I think we need to pass the AZ

Reply:
Well, I think we need to refine the message.

But if the user selects a machine type that is available in one AZ in a region but not in another AZ, it should have that info.

At least for this PR.

Basically

Reply:
I like the message that Andrea suggested, with a slight tweak:

`Machine type '%s' not found in availability zone '%s'. Available machine type list: %s"`

That list might be fairly long, though.